src/erl/clj_module.erl

```diff
@@ -448,29 +437,6 @@ code_change(_Msg, _From, State) ->
 %% Helper Functions
 %%------------------------------------------------------------------------------
 
--spec delete_module(module()) -> ok.
-delete_module(ModuleName) ->
-  case clj_utils:ets_get(?MODULE, ModuleName) of
-    ?NIL -> ok;
-    Module ->
-      ets:delete(Module#module.mappings),
```

Comment:
All of these are gone when the compiling process dies since the module is created in that process.